pyuvdata/uvcal/uvcal.py

```diff
@@ -1264,6 +1682,374 @@ def _check_freq_spacing(self, *, raise_errors=True):
             raise_errors=raise_errors,
         )
 
+    def _add_phase_center(
```

Comment:
This (and similar methods here and on uvdata) might be a good use case for using the docstring parser stuff we've been using for the `read` methods to avoid a lot of docstring repetition.

Reply:
@bhazelton -- finally had a chance to look at this. In _general_ I'd agree, but if I use the docstring parser, it requires importing the UVData object into the UVCal code, which links the two classes in a way that may not be particularly desirable (at the very least, it runs potential risk of setting up a circular dependency). I'd favor keeping this duplicated for now, and then when we eventually migrate the phase catalog entries to proper objects, making the effort then to unify the docstrings (a la Tolkien), but we can talk about this further on the next call.